kratos/python/add_model_part_to_python.cpp

```diff
@@ -1016,8 +1026,10 @@ void AddModelPartToPython(pybind11::module& m)
         .def("GetNodalSolutionStepTotalDataSize", &ModelPart::GetNodalSolutionStepTotalDataSize)
         .def("OverwriteSolutionStepData", &ModelPart::OverwriteSolutionStepData)
         .def("CreateNewNode", ModelPartCreateNewNode)
-        .def("CreateNewElement", ModelPartCreateNewElement)
-        .def("CreateNewCondition", ModelPartCreateNewCondition)
+        .def("CreateNewElement", ModelPartCreateNewElement1)
```

Comment:
Please use lamndas

Reply:
@loumalouomega is this how you inteded the lambda functions to be used?